app/src/main/kotlin/com/wire/android/ui/common/textfield/WireTextField.kt

```diff
@@ -79,7 +79,12 @@ internal fun WireTextField(
             Label(labelText, labelMandatoryIcon, state, interactionSource, colors)
         BasicTextField(
             value = value,
-            onValueChange = onValueChange,
+            onValueChange = {
+                onValueChange(
+                    if(singleLine || maxLines == 1) it.copy(it.text.replace("\n", ""))
+                    else it
+                )
```

Comment:
Can the same thing be achieved by changing the EditText to have only 1 max line

Reply:
It has been like that already, because by default:
```
singleLine: Boolean = true,
maxLines: Int = 1,
```
but you could still enter the new line character 🤷